Jenkinsfiles/Jenkinsfile.cbc-run-multi-pr-checks-w-selenium

```diff
@@ -55,8 +55,7 @@ pipeline {
         sh """
           python -m venv venv
           . venv/bin/activate
-          pip install selenium pytest Flask Flask-WTF
-          pip install cryptography==42.0.4

```

Comment:
This was probably to enforce minimal installs at this stage, but the maintenance overhead of all these extra pinned versions and pip installs is probably too high.